workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/EditTextsTest.kt

```diff
@@ -0,0 +1,108 @@
+package com.squareup.workflow1.ui
+
+import android.text.Selection
+import android.text.SpannableStringBuilder
+import android.widget.EditText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.fail
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@OptIn(WorkflowUiExperimentalApi::class)
+@RunWith(AndroidJUnit4::class)
+class EditTextsTest {
+
+  private val instrumentation = InstrumentationRegistry.getInstrumentation()
+  private val editText = EditText(instrumentation.context)
```

Comment:
whoa

Reply:
I had no idea that we could test isolated views his way.

Should be doing this to test `ViewRegistry` _much_ better.

Reply:
Neither did I, really didn't expect this to work lol.